dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ServiceConfig.java

```diff
@@ -261,21 +263,38 @@ protected void exported() {
         exportedURLs.forEach(url -> {
             if (url.getParameters().containsKey(SERVICE_NAME_MAPPING_KEY)) {
                 ServiceNameMapping serviceNameMapping = ServiceNameMapping.getDefaultExtension(getScopeModel());
-                try {
-                    boolean succeeded = serviceNameMapping.map(url);
-                    if (succeeded) {
-                        logger.info("Successfully registered interface application mapping for service " + url.getServiceKey());
-                    } else {
-                        logger.error(CONFIG_SERVER_DISCONNECTED, "configuration server disconnected", "", "Failed register interface application mapping for service " + url.getServiceKey());
-                    }
-                } catch (Exception e) {
-                    logger.error(CONFIG_SERVER_DISCONNECTED, "configuration server disconnected", "", "Failed register interface application mapping for service " + url.getServiceKey(), e);
-                }
+                ScheduledExecutorService scheduledExecutor = getScopeModel().getBeanFactory()
+                    .getBean(FrameworkExecutorRepository.class).getSharedScheduledExecutor();
```

Comment:
How about put this `ScheduledExecutorService scheduledExecutor = getScopeModel().getBeanFactory()
                    .getBean(FrameworkExecutorRepository.class).getSharedScheduledExecutor();` snippet inside `mapServiceName`, because I see it's only used by `mapServiceName`.